src/surface/transceiver/test/test_parser.py

```diff
@@ -39,22 +44,87 @@ def test_message_parser(packet_handler: SerialReaderPacketHandler) -> None:
+            5.231683254241943,
+            5.247383117675781,
+            5.263083457946777,
+            5.278783321380615,
+            5.294483184814453,
+            5.310183525085449,
+            5.325883388519287,
+            5.341583251953125,
+            5.357283115386963,
+            5.372983455657959,
+            5.388683319091797,
+            5.404383182525635,
+            5.420100212097168,
+            5.435766696929932,
+            5.4514665603637695,
+            5.467166900634766,
+            5.4828667640686035,
+            5.498566627502441,
+            5.514266490936279,
+            5.529966831207275,
+            5.545666694641113,
+            5.561366558074951,
+            5.577066898345947,
+            5.592766761779785,
+            5.608466625213623,
+            5.624216556549072,
+            5.639916896820068,
+            5.655616760253906,
+            5.671316623687744,
+            5.687016487121582,
+            5.702716827392578,
+        ],  # noqa 501
+        depth_data=[
+            0.3828616142272949,
+            0.38326960802078247,
+            0.38755351305007935,
+            0.41376692056655884,
+            0.46119585633277893,
+            0.5239244699478149,
+            0.605522632598877,
+            0.7151702046394348,
+            0.8509292006492615,
+            1.0121876001358032,
+            1.0288132429122925,
+            1.0424809455871582,
+            1.043704867362976,
+            1.0451328754425049,
+            1.0457448959350586,
+            1.0463569164276123,
+            1.0466628074645996,
+            1.0469688177108765,
+            1.0466628074645996,
+            1.0465608835220337,
+            1.047070860862732,
+            1.0463569164276123,
+            1.0458468198776245,
+            1.0457448959350586,
+            1.0444189310073853,
+            1.0446228981018066,
+            1.0451328754425049,
+            1.0442149639129639,
+            1.044724941253662,
+            1.0454388856887817,
+            1.04594886302948,
+        ],  # noqa 501
     )
 
-    with pytest.raises(ValueError, match="Packet expected 3 sections, found 2 sections"):
+    with pytest.raises(ValueError, match='Packet expected 3 sections, found 2 sections'):
         packet_handler.message_parser(NOT_THREE_SECTIONS)
 
-    with pytest.raises(ValueError, match="Packet header length of 3 expected found 2 instead"):
+    with pytest.raises(ValueError, match='Packet header length of 3 expected found 2 instead'):
         packet_handler.message_parser(HEADER_TWO_ELEMENTS)
 
 
 def test_handle_ros_single(packet_handler: SerialReaderPacketHandler) -> None:
     test_queue: Queue[float] = Queue(5)
 
-    for ros_single, test_value, expected_value in zip(ROS_SINGLE_MSGS, TEST_VALUES,
-                                                      EXPECTED_VALUES):
+    for ros_single, test_value, expected_value in zip(
+        ROS_SINGLE_MSGS, TEST_VALUES, EXPECTED_VALUES, strict=False
```

Comment:
If  ROS_SINGLE_MSGS, TEST_VALUES, and EXPECTED_VALUES are all the same lengths, use `strict=True`
[https://docs.python.org/3/library/functions.html#zip](https://docs.python.org/3/library/functions.html#zip)